### PR TITLE
replace `which truffle` with `show-truffle-env`

### PR DESCRIPTION
**Pull request by lsqproduction**

### CODE REVIEW

**Review by cds-amal — CHANGES_REQUESTED** [review]
Thanks @lsqproduction. One nit: the output of **show-truffle-env** and **which truffle** are different and should be updated

File: README.md [hunk 1]
```diff
@@ -179,8 +179,15 @@ Invoke the truffle version you're currently developing.
...
-truffle: aliased to node /home/amal/work/truffle/packages/core/cli.js
+$ show-truffle-env
+truffle: aliased to node /Users/liangliang/code/truffle/packages/core/cli.js
+db-kit: db-kit not found
+
+Truffle v5.5.32 (core: 5.5.32)
+Ganache v7.4.3
+Solidity - 0.8.13 (solc-js)
+Node v16.16.0
+Web3.js v1.7.4e/cli.js
```
Comment (cds-amal) [inline README.md:190]: typo?
```suggestion
Web3.js v1.7.4
```

**Review by cds-amal — APPROVED** [review]
I left a suggested typo fix; otherwise this looks good. Thanks!

**Comment by lsqproduction** [comment]
thank you @cds-amal

**Pull request merged**